Update code conduct (#14)
UBC-MDS/eda_utils_py: CONDUCT.rst

Code of Conduct
---------------

Purpose and principles
^^^^^^^^^^^^^^^^^^^^^^

> This document regulates how team-members should behavior throughout the project. We acknowledge that we should strictly follow `UBC <https://students.ubc.ca/campus-life/student-code-conduct>`_ and `MDS <https://ubc-mds.github.io/resources_pages/code_of_conduct/>`_ rules and codes of conduct. 

Reviewer: ```suggestion
This document regulates how team-members should behave throughout the project. We acknowledge that we should strictly follow `UBC <https://students.ubc.ca/campus-life/student-code-conduct>`_ and `MDS <https://ubc-mds.github.io/resources_pages/code_of_conduct/>`_ rules and [Code of Conduct](https://github.com/UBC-MDS/eda_utils_py/blob/main/CONDUCT.rst). 
```

We see the project as an opportunity to discuss real-world issues through the lens of data science and analytics and believe that the freedom of thoughts and speech is essential to the success of this project, and higher education in general. 
We encourage a diversity of perspectives and opinions, and candid expression and civil competition thereof, especially regarding public and social issues that potentially will become factors to be considered in the project. 

We appreciate the diversity of team members' personalities and backgrounds. Their gender, ethnicity, national origin, sexual orientation, and religion should be respected.   

Encouraged and untolerated behaviors 
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

- As stipulated by UBC and MDS rules, the following behaviors are encouraged: 

1. Use welcoming and polite language
2. Be respectful of different viewpoints and experiences
3. Gracefully accept constructive criticism
4. Focus on what is best for the community
5. Show courtesy and respect towards other community members
6. Be honest about work progress and difficulties 

- As stipulated by UBC and MDS rules, the following behaviors will not be untolerated: 

> 1. written or verbal personal attacks on people based on their social and political belief, religion, gender, ethnicity, national origin, sexual orientation

Reviewer: ```suggestion
1. Written or verbal personal attacks on people based on their social and political belief, religion, gender, ethnicity, national origin, sexual orientation
```

> 2. causing someone to fear for their safety, such as through stalking, following, or intimidation

Reviewer: ```suggestion
2. Causing someone to fear for their safety, such as through stalking, following, or intimidation
```

> 3. violent threats or language directed against another person

Reviewer: ```suggestion
3. Violent threats or language directed against another person
```

> 4. the display of sexual or violent images

Reviewer: ```suggestion
4. The display of sexual or violent images
```

> 5. unwelcome sexual attention and nonconsensual or unwelcome physical contact

Reviewer: ```suggestion
5. Unwelcome sexual attention and nonconsensual or unwelcome physical contact
```

> 6. sustained disruption of talks, events, or communications

Reviewer: ```suggestion
6. Sustained disruption of talks, events, or communications
```

> 7. insults or put-downs

Reviewer: ```suggestion
7. Insults or put-downs
```

> 8. excessive swearing

Reviewer: ```suggestion
8. Excessive swearing
```

> 9. incitement to violence, suicide, or self-harm

Reviewer: ```suggestion
9. Incitement to violence, suicide, or self-harm
```

> 10. continuing to initiate interaction (including photography or recording) with someone after being asked to stop publication of private communication without consent

Reviewer: ```suggestion
10. Continuing to initiate interaction (including photography or recording) with someone after being asked to stop publication of private communication without consent
```
